src/resource/solidDataset.ts

```diff
@@ -1133,3 +1137,37 @@ function resolveLocalIrisInThing(
       : thing.url,
   });
 }
+
+/**
+ * Fetch the contents of '.well-known/solid' for a given resource URL.
```

Comment:
Should we extend the documentation a little bit to mention briefly what the well-known document is ?

Reply:
Do we have a description of that somewhere already? I've had a little look and not turned anything up.

Happy to expand this a bit, but suspect I might miss some conceptual nuance.